### ISSUE TITLE
Some issues with solutions and projects

### ISSUE THREAD

**Issue by ThomasBarnekow** [EricWhiteDev/OxPt#2]
I've noted a few issues with the solutions and projects:

The names are not clear enough, meaning you can only guess what is meant by SIP (System.IO.Packaging) and OS (possibly Open XML SDK, once you open the solution and find that it references a project which it can't find, though).

It looks like OpenXmlPowerToolsOS references a project sitting in an absolute path (C:\Visual Studio 2010\Open-Xml-Sdk\Open-XML-SDK.csproj) in which people might not have their projects. Therefore, that solution won't load correctly for most users.

OpenXmlPowerToolsSIP.csproj references the System.IO.Packaging.dll which, however, it will likely not find in many cases. Here's the reference from the .csproj file:
```
<Reference Include="System.IO.Packaging">
    <HintPath>..\..\..\Open-Xml-Sdk\build\OpenXmlSdkLib\Debug\System.IO.Packaging.dll</HintPath>
</Reference>
```
This tries to find a DLL in the `build` folder on the solution level. However, after having reorganized the solution and projects, that build folder will no longer contain the latest DLLs.

We could simplify this by:

* having a single solution with multiple projects, each defining the required .NET Framework and references and each referencing the same source files; and

* including the required libraries (e.g., Open-XML-SDK.dll, System.IO.Packaging.dll) in a Lib folder.

**Comment by EricWhiteDev** [comment]
Yes, you are right, the projects need to be rationalized, and if possible, merged so that there is one project with multiple build configurations.  Also will include the DLLs in a Lib folder.
